app/editor/src/features/admin/reports/constants/defaultTemplate.ts

```diff
@@ -0,0 +1,35 @@
+export const defaultTemplate = `@inherits RazorEngineCore.RazorEngineTemplateBase<TNO.Services.Reporting.Models.TemplateModel>
```

Comment:
@CelloLiu I've added this to make it easier to start a new report.  This provides a standard template that will obey the various configuration rules.  I still need to handle more configuration options, but this is a start.